src/components/list/ItemsList.tsx

```diff
@@ -59,7 +59,7 @@ export default class ItemsList<T extends ListItem> extends React.Component<Props
             {this.props.renderItem(item, selectedIds.has(item.id as string))}
           </TouchableOpacity>
         )}
-        keyExtractor={(item) => item.id as string}
+        keyExtractor={(item, index) => item.id.toString() + index.toString()}
```

Comment:
It's the only way I found to make it unique. With only the id, the cars list seems to have duplicates. And with only the index, I have issues when searching (e.g in the users list)

Reply:
Cars have unique IDs otherwise this is a bug.
Let me know.
index should not be used as when data is bound again you break the way the differential rendering works in react-native.

Reply:
Ok the problem sis not specific to the cars apparently. 
Just tried with the id only and i got this warning.
<img width="380" alt="Screenshot 2021-04-01 at 11 37 21" src="https://user-images.githubusercontent.com/32574035/113275295-ec501b80-92de-11eb-8edd-7e39d286dc19.png">

The same warning on tags and big issues when scrolling a bit fast (blank areas and flickering)

<img width="443" alt="Screenshot 2021-04-01 at 11 38 17" src="https://user-images.githubusercontent.com/32574035/113275469-202b4100-92df-11eb-83a8-67cea1e688a8.png">